CHANGELOG.md

```diff
@@ -3,6 +3,16 @@ All notable changes to this project will be documented in this file.
 
 ## Unreleased
 
+### Changed
+
+ - Minimum rust version is now 1.56
+ - C++ compiler requires C++20
+ - In the C++ interpreter API `std::span` is used for callbacks arguments, instead of `sixtyfps::Slice`
```

Comment:
This is a good example of something where we should start documenting it. $customer is using `invoke_callback` with parameters, for example.

We can do that separately from this merge request though.